### PR TITLE
chore: remove all package.json from v-build ownership to speed-up PR review velocity

### PR DESCRIPTION
**Pull request by Hotell**
Based on v-build agreement removing all package.json files from its ownership and keeping only all common config files

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/854d1159c9a57d496f6db1ab9f20885136d20cc6'>854d1159c9a57d496f6db1ab9f20885136d20cc6</a></sub>
<div id="bundleSizeComment9423"/>

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -272,7 +272,7 @@ packages/react-experiments/src/components/TileList @ThomasMichon
 **/tsconfig.json @microsoft/fluentui-react-build
 **/tsconfig.lib.json @microsoft/fluentui-react-build
 **/tsconfig.spec.json @microsoft/fluentui-react-build
-**/package.json @microsoft/fluentui-react-build
+# **/package.json @microsoft/fluentui-react-build
```
Comment (khmakoto) [inline .github/CODEOWNERS:275]: This one should probably be removed, not only commented.

**Comment by Hotell** [comment]
admin merged as there was issue running github actions

**Pull request merged**